scripts/xtensa-build-zephyr.py

```diff
@@ -797,7 +797,12 @@ class InstFile:
 		# Remove it like some gcc test scripts do:
 		# https://gcc.gnu.org/git/?p=gcc.git;a=commit;h=c7046906c3ae
 		if "strip" in str(dstname):
-			execute_command([str(x) for x in [objcopy, "--remove-section", ".comment", src, dst]])
+			execute_command(
+				[str(x) for x in [objcopy, "--remove-section", ".comment", src, dst]],
+				# Some xtensa installs don't have a
+				# XtensaTools/config/default-params symbolic link
```

Comment:
Not sure the comment is really needed, the vendor toolchain passes junk in the environment.  No reason to expect the binutils equivalents to work any differently from the compiler.

Reply:
Thanks for the review!

> The toolchain install process in xtensa is really fragile. It's almost enough to just tar up and copy the directory around, but some things break.

While investigating #7114 I had the... "opportunity" to compare installation across different machines. All the binaries are (fortunately!) identical and the only differences are absolute paths in a number of text config files (set by the installer script). It's much simpler than what I feared initially.

>  Not sure the comment is really needed, the vendor toolchain passes junk in the environment. No reason to expect the binutils equivalents to work any differently from the compiler.

I've successfully invoked `xt-objdump` many times without any `XTENSA_` variable thanks to this  `default-params` symlink - without realizing it until today! I think that symlink is worth knowing. Here for a lack of a better place.